activators/caila/src/main/kotlin/com/justai/jaicf/activator/caila/Duckling.kt

```diff
@@ -0,0 +1,12 @@
+package com.justai.jaicf.activator.caila
```

Comment:
I'm not sure that we have to specify CAILA system entities in our sources. It's a nice to have object, but it may mislead user into thinking that CAILA can use only these entities, which is not true.

Reply:
I can remove this, but there are many such objects already in JAICF.
Like `TelegramEvent`, `DialogflowIntent`, etc.